src/Microsoft.Azure.WebJobs.Extensions.Kafka/Diagnostics/ActivityProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Executors;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.Diagnostics
+{
+    internal class ActivityProvider
```

Comment:
scope is internal are we going to use this only within this namespace ??

Reply:
I have changed the namespace to Microsoft.Azure.WebJobs.Extensions.Kafka.